docs/janssen-server/developer/customization/customize-web-pages.md

```
@@ -163,22 +163,22 @@ reference it in the custom script as follows:
 [built-in resource bundles](https://github.com/JanssenProject/jans/blob/main/jans-auth-server/server/src/main/resources/).
 
 To override the defaults, the custom `.properties` files should be uploaded
-as [custom assets](#managing-custom-assets). If the file name matches the 
+as [custom assets](#managing-custom-assets). If the file name matches the
 existing resource bundle, then the custom bundle will override the built-in
-resource bundle. 
+resource bundle.
 
 Examples of file names are:
 
  * jans-auth_en.properties
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)